### PR TITLE
Get passengers for Users + Tests PR

### PR DESCRIPTION
**Pull request by emrebilge**
Not sure if I am doing the testing correctly

### CODE REVIEW

File: build/controllers/Passenger.controller.js [hunk 1]
```diff
@@ -47,15 +86,44 @@ exports.getAllPassengersForUser = getAllPassengersForUser;
...
  */
-const getPassengerById = (req, res) => __awaiter(void 0, void 0, void 0, function* () {
-    // get the passengerId from the query parameters
-    // const { passengerId } = req.query;
-    // create a fake passenger
-    const passenger = (0, test_data_1.createTestPassengerData)();
-    // return the passenger
-    res.status(200).send(passenger);
-});
+// Function to get a passenger by their passengerId
```
Comment (begzod03) [inline build/controllers/Passenger.controller.js]: Nice comment

File: package.json [hunk 2]
```diff
@@ -5,7 +5,7 @@
   "main": "index.ts",
   "scripts": {
     "start": "nodemon src/index.ts",
-    "test": "ENVIRONMENT=test nyc mocha -r dotenv/config --timeout 20000 --exit -r ts-node/register src/tests/*.tests.ts",
+    "test": "cross-env ENVIRONMENT=test nyc mocha -r dotenv/config --timeout 20000 --exit -r ts-node/register src/tests/*.tests.ts",
```
Comment (jacoblurie29) [inline package.json:8]: What is this?

File: package.json [hunk 3]
```diff
@@ -28,6 +28,7 @@
     "nodemon": "^3.0.1",
     "pino": "^8.17.2",
     "pino-pretty": "^10.3.1",
+    "sinon": "^17.0.1",
```
Comment (jacoblurie29) [inline package.json:31]: What is this?

File: package.json [hunk 4]
```diff
@@ -42,6 +43,7 @@
     "@typescript-eslint/parser": "^6.11.0",
     "chai": "^4.3.10",
     "chai-http": "^4.4.0",
+    "cross-env": "^7.0.3",
```
Comment (jacoblurie29) [inline package.json:46]: What is this?

File: src/controllers/Passenger.controller.ts [hunk 5]
```diff
@@ -63,20 +103,42 @@ export const getPassengerById = async (req: Request, res: Response) => {
...
+  };
+
+  try {
+    // const createdRecords = await base('Passengers').create([newRecord]);
+    // const createdPassenger = createdRecords[0];
+    // res.status(200).send({ createdPassenger });
+  } catch (error) {
+    logger.info(error);
+    res.status(500).send({ error: 'Failed to create passenger' });
+  }
```
Comment (jacoblurie29) [inline src/controllers/Passenger.controller.ts:141]: Can you comment this out

File: src/tests/PassengerController.tests.ts [hunk 6]
```diff
@@ -0,0 +1,104 @@
...
+    // Close the server after tests
+    server.close(() => {
+      console.log('Test server closed');
+      done();
+    });
+  });
+
+  beforeEach(() => {
+    // Stub the Airtable base method to prevent actual API calls
+    airtableBaseStub = sinon.stub(Airtable.prototype, 'base').callsFake(() => ({
```
Comment (jacoblurie29) [inline src/tests/PassengerController.tests.ts:35]: what is this

**Review by jacoblurie29 — CHANGES_REQUESTED** [review]

**Review by jacoblurie29 — APPROVED** [review]
LGTM

**Pull request merged**